app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for character in phrase:
+        if character == letter:
```

Comment:
The function is supposed to be case insensitive, but the current implementation does not handle case sensitivity. Consider converting both 'character' and 'letter' to the same case (e.g., lower case) before comparison.